docs/variants/msi_z690/firmware-update.md

```diff
@@ -12,6 +12,46 @@ sudo dmidecode -t bios | grep Version
 
 Alternatively, it can be checked in the `BIOS Setup Menu`.
 
+## Prerequisites
+
+Before proceeding, ensure that the firmware protections are disabled in
+[Dasharo Security Options](../..//dasharo-menu-docs/dasharo-system-features.md).
+Both `BIOS boot medium lock` and `Enable SMM BIOS write protection` should be
+unchecked. [UEFI Secure Boot](../../dasharo-menu-docs/device-manager.md#secure-boot-configuration)
+must be disabled as well (uncheck `Attempt Secure Boot` if
+`Current Secure Boot State` does not say `Disabled`). To apply changes you will
+need to reboot.
+
+You can use [Dasharo Tools Suite (DTS)](../../dasharo-tools-suite/releases.md)
+or your own Linux distribution for firmware update. You will need a correct
+flashrom application for that purpose. Linux distributions may not yet have the
+support for the newesy chipsets in flashrom installed via package manager.
+Procedure for building the right flashrom is described in Build flashrom
+section in the [Initial deployment documentation](initial-deployment.md#initial-deployment-manually).
+DTS is already equipped with a correct flashrom build. You will need to
+download a firmware update binary. When you boot DTS enter the shell from the
```

Comment:
I would rephrase whole paragraph to make it very clear. Now it say DTS, Linux distro, DTS - no easy to follow. Something as follows (your content can be easily reused):

Please use one of the following environments to perform Dasharo update:
- DTS - ...
- Linux distribution of your choice  - ...